### ISSUE TITLE
Accessibility: user cannot cancel NDDB prompt

### ISSUE THREAD

**Issue by tierninho** [desktop/desktop#6465]
User has no way to tab over to the `X` button to cancel the NDDB prompt.

<img width="327" alt="screen shot 2018-12-20 at 11 29 51 am" src="https://user-images.githubusercontent.com/14828183/50306562-c58db400-044a-11e9-9e1f-95e96d136e4a.png">

v1.5.1, mac

**Comment by tierninho** [comment]
Closing this as it has not come up enough times to be validated as a real problem.
